### DISCUSSION TITLE
SSE Enable without an external KMS Service

### DISCUSSION THREAD

**Discussion by shahulpm** [minio/minio#19012] [Q&A] [answered]
Is That possible to enable SSE without an External KMS service, If That, so please provide documentation. I couldn't find any relevant documentation

**Comment by harshavardhana** [comment, marked as answer]
Please use an external KMS for production. Use KMS_SECRET_KEY for local testing

**Comment by alxndrsn** [comment]
The following env vars seem to do it:

* `MINIO_KMS_AUTO_ENCRYPTION`
* `MINIO_KMS_SECRET_KEY`

e.g.

```sh
MINIO_KMS_AUTO_ENCRYPTION=on
MINIO_KMS_SECRET_KEY="some-key-name:$(head -c32 /dev/urandom | base64)"
```

I can't find a link to good docs for this.
